sdk/core/core-amqp/package.json

```diff
@@ -72,7 +72,7 @@
     "rhea-promise": "^0.1.15"
   },
   "devDependencies": {
-    "@azure/identity": "^0.1.0",
+    "@azure/identity": "^1.0.0-preview.1",
```

Comment:
Latest news from version land is to use `-preview.1.0` suffix in the end.

This is so that we can ship bug fixes, but still call the library "Preview 1" until we actually get to "Preview 2"

Reply:
Sounds like a reasonable plan, I'll change it here and in the `core-http` PR.